dipy/reconst/dti.py

```diff
@@ -1993,7 +2002,65 @@ def design_matrix(gtab, dtype=None):
     return -B
 
 
-def quantize_evecs(evecs, odf_vertices=None):
+def _quantize_evecs_parallel(data, odf_vertices, v, nbr_processes):
+    if nbr_processes is -1:
```

Comment:
Should be `if nbr_processes == -1:` - although CPython does keep an interned list of singleton small integers, so this check (equivalent to `id(-1) == id(-1)`) will generally work, it's an implementation detail - see http://stackoverflow.com/questions/2988017/string-comparison-in-python-is-vs#2988117 for discussion.